matching-service/index.js

```diff
@@ -34,6 +43,11 @@ const router = express.Router()
 
 router.post('/', findMatch)
 router.delete('/', deleteMatch)
+router.get('/rooms', findAllRooms)
+router.get('/room', findOneRoom)
+router.post('/room', createRoom)
+router.patch('/room/:room_id', getRoomByRoomId ,updateRoom)
+router.delete('/room/:room_id', getRoomByRoomId, deleteRoom)

```

Comment:
Hmm I don't think we need to fetch the room before we delete it... we can just use mongoose's delete operation (https://mongoosejs.com/docs/models.html#deleting) and pass in the ID. That way we don't need the extra roundtrip query to mongoDB

Reply:
The issue is I don't know the mongodb ID

Reply:
Ok! 👍🏻